website/docs/api/jestPreviewConfigure.md

```diff
@@ -59,3 +59,9 @@ Default: `false`
 Automatically preview the UI in the external browser when the test fails. You don't need to invoke `preview.debug()` by yourself anymore.
 
 Set to `false` if you experience any error or just want to opt out.
+
+## sharedSassResources: string[]
+
+Default: `undefined`
```

Comment:
The default value should be [], just for consistency with sassLoadPaths, and no need to check that `sharedSassResources` is existed. Also, I think we can move this doc right below the `sassLoadPaths` doc, so we wrap all the `sass` configs in the same area.